com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transaction/ZigBeeTransactionManager.java

```diff
@@ -330,6 +332,9 @@ public ZigBeeTransactionProfile getBroadcastProfile() {
      * @param command the {@link ZigBeeCommand} to send
      */
     public void sendTransaction(ZigBeeCommand command) {
+        if (isShutdown) {
+            return;
```

Comment:
I wonder if we should log this on debug or trace level, to see if someone is still trying to use a shutdown transactionmanager. WDYT?

Reply:
Ok, I'll update this.